vic/drivers/cesm/bld/vic.buildexe.csh

```diff
@@ -0,0 +1,23 @@
+#! /bin/csh -f
+
+cd $OBJROOT/lnd/obj
+
+set comp = 'unknown'
+if ($COMP_INTERFACE == 'MCT' ) set comp = mct
+if ($COMP_INTERFACE == 'ESMF') set comp = esmf
+
+cat >! Filepath << EOF
+$CASEROOT/SourceMods/src.vic
+$CODEROOT/lnd/vic/vic
+$CODEROOT/lnd/vic/vic/vic_run/src
+$CODEROOT/lnd/vic/vic/vic_run/include
+$CODEROOT/lnd/vic/vic/drivers/shared/src
+$CODEROOT/lnd/vic/vic/drivers/shared/include
+$CODEROOT/lnd/vic/vic/drivers/cesm/src
+$CODEROOT/lnd/vic/vic/drivers/cesm/include
+$CODEROOT/lnd/vic/vic/drivers/cesm/cpl_$comp/
+EOF
+
+set vicdefs = ""
```

Comment:
You are going to have to push that change into the Macros file and the options will probably depend on the compiler.  You will modify scripts/ccsm_utils/Machines/config_compilers.xml.  Have a look there and see if you can figure it out.  I don't think there is much documentation unfortunately.  there might need to be some trial and error.  In the end, you want something like 
  <CFLAGS MODEL="vic"> -ggdb -O0 -Wall -Wextra -fPIC </ADD_CFLAGS>
somewhere and you want the Macros file to end up with something like
ifeq ($(MODEL), vic) 
   CFLAGS += --ggdb -O0 -Wall -Wextra -fPIC
endif